composer.json

```diff
@@ -11,17 +11,34 @@
             "homepage": "https://github.com/symfony-cmf/ContentBundle/contributors"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/ElectricMaxxx/RestBundle.git"
+        },
+        {
+            "type": "path",
+            "url": "https://github.com/dantleech/Sylius/tree/phpcrodm_form_builder/src/Sylius/Bundle/ResourceBundle",
```

Comment:
I don't know if there is one. You can just run the tests locally use a symlink -
if this PR depends on that PR, then it can't be merged until the the
other one is merged anyway :)

On Mon, May 30, 2016 at 06:03:34AM -0700, Maximilian Berghoff wrote:
>    In [1]composer.json:
> 
>  > @@ -11,17 +11,34 @@
>  >              "homepage": "https://github.com/symfony-cmf/ContentBundle/contributors"
>  >          }
>  >      ],
>  > +    "repositories": [
>  > +        {
>  > +            "type": "vcs",
>  > +            "url": "https://github.com/ElectricMaxxx/RestBundle.git"
>  > +        },
>  > +        {
>  > +            "type": "path",
>  > +            "url": "https://github.com/dantleech/Sylius/tree/phpcrodm_form_builder/src/Sylius/Bundle/ResourceBundle",
> 
>    [2]@dantleech you know what would be the right solution. Tried several
>    options:
> 
>      * vcs+ "*.git" to get get sylius, to simply copy => version problem as
>        sylius requires ^1.2
>      * path + branch path to ResourceBundle (current one) with your branch to
>        overwrite the "sylius/resource-bundle" but does not work do not get
>        your branch
> 
>    —
>    You are receiving this because you were mentioned.
>    Reply to this email directly, [3]view it on GitHub, or [4]mute the thread.
> 
>    Reverse link: [5]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/symfony-cmf/content-bundle/pull/144#discussion_r65068806
>    2. https://github.com/dantleech
>    3. https://github.com/symfony-cmf/content-bundle/pull/144/files/6940d5cac5884ee7818db1346d3ffc8de5ed1f3b#r65068806
>    4. https://github.com/notifications/unsubscribe/AAgZceL4GJi_fHkaM9Xp72wNi9_5CNvVks5qGuAmgaJpZM4IWrft
>    5. https://github.com/symfony-cmf/content-bundle/pull/144/files/6940d5cac5884ee7818db1346d3ffc8de5ed1f3b#r65068806